File: gherkin/java/src/main/java/gherkin/pickles/PickleCell.java
package gherkin.pickles;

public class PickleCell {
    private final PickleLocation location;
    private final String value;

    public PickleCell(PickleLocation location, String value) {
        this.location = location;
        this.value = value;
    }

    public String getValue() {
        return value;
    }

    public PickleLocation getLocation() {
        return location;
    }
}

File: gherkin/dotnet/Gherkin/Pickles/PickleCell.cs
using System.Runtime.Serialization;

namespace Gherkin.Pickles
{
    public class PickleCell
    {
        [DataMember(Name = "value")]
        public string Value { get; private set; }

        public PickleCell(string value)
        {
            Value = value;
        }
    }
}
